lib/reporting/reports/suppliers/helpers/columns_helper.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+module Reporting
+  module Reports
+    module Suppliers
+      module Helpers
+        module ColumnsHelper
+          include LineItemsAccessHelper
+
+          def producer
+            proc { |line_items| supplier(line_items).name }
+          end
+
+          def producer_address
+            proc { |line_items| supplier(line_items).address&.full_address }
+          end
+
+          def producer_abn_acn
+            proc do |line_items|
+              supplier = supplier(line_items)
+
+              supplier.abn.presence || supplier.acn
```

Comment:
I'm not sure if the requirement has been clarified already, but I would think it's better to include both. 
For example separated by a slash like so:
```suggestion
              [supplier.abn, supplier.acn].compact_blank.join("/")
```

Reply:
It's fixed here: 749c00312e180c686699557587993cf9e7c2ff6a